### PR TITLE
Enhance CI

### PR DESCRIPTION
**Pull request by szepeviktor**
Fails as expected.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,10 +7,13 @@ jobs:
...
+    - name: Check Composer configuration
+      run: composer validate --strict
     - name: Install dependencies
       run: composer install
     - name: Start Redis
       uses: supercharge/redis-github-action@1.1.0
-    - name: Run tests
-      run: phpunit
\ No newline at end of file
+    - name: Run unit tests
```
Comment (stancl) [inline .github/workflows/ci.yml]: They're not exactly unit tests. They for example execute the queue worker to run the queued jobs.

Revert this one change and I'll merge.

Thanks.

Comment (szepeviktor) [reply .github/workflows/ci.yml]: PHPUnit running non-unit tests??

**Comment by stancl** [comment]
Yes. These are feature tests in my definition.

**Comment by stancl** [comment]
Can you make the tests pass now that we see that they find issues?

**Comment by szepeviktor** [comment]
Done. 🍏

**Comment by stancl** [comment]
Thanks!

**Comment by szepeviktor** [comment]
You're welcome.

**Comment by stancl** [comment]
Actually going to revert this. composer.lock files don't belong into libraries.

**Comment by szepeviktor** [comment]
FYI `validate` also checks the json file.

**Pull request merged**